lisa/tools/windows_feature.py

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import Any, List
+
+from lisa.executable import Tool
+from lisa.tools.powershell import PowerShell
+
+
+# WindowsFeature management tool for Windows.
+# It can install, uninstall, and check the status of Windows features.
+# This tool uses PowerShell to manage Windows features.
```

Comment:
Can you please provide example of 1 or 2 features?

Reply:
Updated it in comments